fractions/cdi-extensions/cdi-jaxrsapi/src/main/java/org/wildfly/swarm/cdi/jaxrsapi/runtime/ClientServiceFactory.java

```diff
@@ -0,0 +1,166 @@
+package org.wildfly.swarm.cdi.jaxrsapi.runtime;
```

Comment:
Ah. Just a side effect of me tracking down a bug that was ultimately in fraction plugin.